src/contrib/parallel-rdp/parallel-rdp-standalone/vulkan/device.hpp

```diff
@@ -506,6 +506,7 @@ class Device
 	                                 VkShaderStageFlagBits stage = VK_SHADER_STAGE_COMPUTE_BIT) const;
 
 	const QueueInfo &get_queue_info() const;
+  	const RenderPass &request_render_pass(const RenderPassInfo &info, bool compatible);
```

Comment:
This was made public in my fork for other reasons that aren't relevant for your emulator